src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -113,10 +119,32 @@ private List<BuildData> GetBuildManifestsMetadata(string manifestsFolderPath)
                             version = string.Empty;
                         }
 
-                        AddAsset(assets, blob.Id, version, manifest.Location, "Container", blob.NonShipping);
+                        AddAsset(
+                            assets, 
+                            blob.Id, 
+                            version, 
+                            manifest.InitialAssetsLocation ?? manifest.Location, 
```

Comment:
> manifest.InitialAssetsLocation ?? manifest.Location,  [](start = 27, length = 54)

In the Packages foreach loop we could only end up having a Container type if InitialAssetsLocation != null but looks like this it does not matter. Is that correct?

Reply:
The blobs will initially be on the Pipeline Artifacts container so.. I didn't have to change the name. But this value might change if we decide to add a new artifact location type - @mmitche @alexperovich